app/modules/assets/resources.py

```diff
@@ -198,7 +198,6 @@ class AssetSrcRawByID(Resource):
         },
     )
     def get(self, asset):
-        log.info(f'Sage raw src read of Asset {asset.guid}')
```

Comment:
We already have a trace line for every raw src read from werkzeug so this was unnecessary duplication.